doc/help/kdb-backup.md

```diff
@@ -1,6 +1,6 @@
 # kdb-backup -- Make a backup of current KDB
 
-## SYNOPSES
+## SYNOPSIS
```

Comment:
Can you please add this spell correction also to scripts/sed?

Reply:
As "synopses" is the plural of "synopsis" this would "correct" usage of "synopses" everywhere, right? Do you still want me to add this?

Reply:
As we do not use ALL-CAPS words outside of man pages and in man pages only SYNOPSIS is used, s/SYNOPSES/SYNOPSIS/g should be safe.